store/engine/consul/consul_test.go

```diff
@@ -47,15 +47,15 @@ func TestBasicOperations(t *testing.T) {
 	}()
 
 	ctx := context.Background()
-	keys := []string{"a/b/c0", "a/b/c1", "a/b/c2"}
+	keys := []string{"/a/b/c0", "/a/b/c1", "/a/b/c2"}
```

Comment:
the error really happened because the `nsPrefix` has a leading `/`, but I think if we just keep the tests consistent with the other engines it'll be ok?